### PR TITLE
javasript week1 homework first commit

### PR DESCRIPTION
**Pull request by Lokesh-sw**

### CODE REVIEW

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:1]: It is good to have a habit to give a meaningful name to files, I wouldn't say 1.js is the right approach regarding naming

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:8]: Check Template literals and how you can write these statements in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:4]: why let?

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:12]: I would use Prettier 🙂

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:15]: why let?

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:19]: `shouldShowResultInDogYearsWe` should be boolean (either true or false), and here it is 10
The assignment says: _add an extra variable called shouldShowResultInDogYears. If it is true we should show the result in dog years, if it is false we should show it in human years._

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:45]: why let? 🙂  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:46]: why let, it should be const?

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:47]: 👍

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:109]: What if you need two random numbers per array? 🙂

File: javascript/javascript1/week1/1.js
Comment (shravanipagilla) [inline javascript/javascript1/week1/1.js:121]: Hej  Lokesh, you did very  hard work  . i felt nothing to change .

**Review by shravanipagilla — COMMENTED** [review]
nice work

File: javascript/javascript1/week1/1.js
Comment (cecastosic) [inline javascript/javascript1/week1/1.js:79]: Have you checked if this code is working? 
It doesn't btw. 
I will paste the corrected version of your homework here but you should be aware that you shouldn't submit the homework that doesn't work 🙂 
```
// for Peter
const estimatedCostPeter = volumeInMeters[0] * 2.5 * 1000 + gardenSizeInM2[0] * 300;
if (estimatedCostPeter < houseCost[0]) {
  console.log(`Peter is paying ${houseCost[0] - estimatedCostPeter} extra`);
} else if (estimatedCostPeter > houseCost[0]) {
  console.log(`Peter saving ${estimatedCostPeter - houseCost[0]}`);
} else {
  console.log("Correct Price");
}

// for Julia
const estimatedCostJulia = volumeInMeters[1] * 2.5 * 1000 + gardenSizeInM2[1] * 300;
if (estimatedCostJulia < houseCost[1]) {
  console.log(`Julia is paying ${houseCost[1] - estimatedCostJulia} extra`);
} else if (estimatedCostJulia > houseCost[1]) {
  console.log(`Julia saving ${estimatedCostJulia - houseCost[1]}`);
} else {
  console.log("Correct Price");
}
```

**Review by cecastosic — COMMENTED** [review]
@Lokesh-sw Please go through my comments and try to understand when you should use `let` and when `const`. Maybe this video can help https://youtu.be/1mgLWu69ijU 
Be careful before submitting the homework, the submitted code should work. Thanks 🌸

File: javascript/javascript1/week1/week1.js
Comment (cecastosic) [inline javascript/javascript1/week1/week1.js:42]:
```suggestion
  console.log(`Peter is saving ${estimatedCostPeter - houseCost[0]}`);
```

File: javascript/javascript1/week1/week1.js
Comment (cecastosic) [inline javascript/javascript1/week1/week1.js:54]:
```suggestion
  console.log(`Julia is saving ${estimatedCostJulia - houseCost[1]}`);
```

Comment (cecastosic) [reply javascript/javascript1/week1/week1.js:54]: You can also notice that you are repeating some of the words ('is saying', 'is paying', 'extra'...)

**Review by cecastosic — APPROVED** [review]
Cool! 🚀 But please remove 1.js file before merging 🙂 and change HTML so it uses the new js file.

File: javascript/javascript1/week1/index.html
Comment (cecastosic) [inline javascript/javascript1/week1/index.html]: You should also change here which file you want to use.

**Pull request merged**
